coreos-base/oem-packet/files/reset-interfaces.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+logger "Hard reset of all network interfaces due to upstream systemd-networkd issue: https://github.com/coreos/bugs/issues/36 ."
+ip link set enp1s0f0 down
+ip link set enp1s0f1 down
+ip link del bond0
+systemctl restart systemd-networkd
+sleep 15
```

Comment:
Adding Wants/After=sys-devices-virtual-net-bond0.device to the next unit should be more reliable.